From 9d632a609d85d1c47f4f62f3cc7837d400d0a055 Mon Sep 17 00:00:00 2001
From: Wei Lee <wtlee@google.com>
Date: Mon, 25 Feb 2019 05:13:10 +0000
Subject: [PATCH] Update the version of Chrome Camera App to 5.4.0

This CL updates the version of Chrome Camera App to 5.4.0.

Bug: 935051
Test: None
Change-Id: I21be6f014584f8ab71cb2f0fb390875547787e0e
Reviewed-on: https://chromium-review.googlesource.com/c/1484970
Reviewed-by: Sheng-hao Tsao <shenghao@google.com>
Commit-Queue: Wei Lee <wtlee@chromium.org>
Cr-Commit-Position: refs/heads/master@{#635032}
---
 chrome/browser/resources/chromeos/camera/src/manifest.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/browser/resources/chromeos/camera/src/manifest.json b/chrome/browser/resources/chromeos/camera/src/manifest.json
index 5805518c302d4..a82dc1abb85be 100644
--- a/chrome/browser/resources/chromeos/camera/src/manifest.json
+++ b/chrome/browser/resources/chromeos/camera/src/manifest.json
@@ -3,7 +3,7 @@
   "manifest_version": 2,
   "name": "__MSG_name__",
   "description": "__MSG_description__",
-  "version": "5.2.6",
+  "version": "5.4.0",
   "default_locale": "en",
   "minimum_chrome_version": "60.0.0.0",
   "icons": {
